assets/stylesheets/global/variables-dark.scss

```diff
@@ -9,7 +9,7 @@ $sidebarMediumWidth: 16rem;
 
 $contentBackground: #2d2d2d;
 
-$textColor: #b4b3af;
+$textColor: #e6e6e5;
```

Comment:
I find this too bright and too close to links (which by the way could you change them to `#fff` [here](https://github.com/dhenson02/devdocs/blob/master/assets/stylesheets/global/variables-dark.scss#L32)).

Does `#c9c9c9` look bright enough on your screen?